frontend/js/helpers/converters.js

```diff
@@ -44,4 +44,9 @@ const rescaleCO2Value = (value,unit) => {
     else if(value > 1_000_000) return [(value/(10**6)).toFixed(2), 'g'];
     else if(value > 1_000) return [(value/(10**3)).toFixed(2), 'mg'];
     return [value.toFixed(2) , unit];
-}
\ No newline at end of file
+}
+
+// Function to format 1000000 to 1,000,000
+const formatLongValue = (value, delimiter=",") => {
```

Comment:
What is this function? Why write such a complex regex if there are functions in the stdlib?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Also this function is buggy. It does what it should for the input but cannot be reused with proper integers / rationals.

Example: `a = 10000000.00; a.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",")`

Reply:
I wasn't aware of this function in the stdlib - that's definitely the better solution. Refactored to remove my custom function and used that instead.